src/main/java/com/easypost/model/BetaPaymentRefund.java

```diff
@@ -0,0 +1,12 @@
+package com.easypost.model;
+
+import java.util.List;
+import lombok.Getter;
+
+@Getter
+public class BetaPaymentRefund extends EasyPostResource{
+    private String refundedAmount;
```

Comment:
Probably, the responses in our internal doc are `string` and `int`, looks like I need to add an attribute for `refunded_amount_currencys`